akirataguchi115/lukuvinkkikirjasto
# deleted .classpath and .project that are gitignored

## Description
deleted .classpath and .project that are gitignored. Both are IDE specific configs

## How to test:
if you don't need .classpath or .project try to remove them and then `git pull` to see if .gitignore worked
